get-dashboard.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+#set -euo pipefail
+set -eu
+
+VERSION="$1"
+
+# ensure dir
+cd -P -- "$(dirname -- "$0")"
+
+DOWNLOAD_URL='https://github.com/emqx/emqx-dashboard-frontend/releases/download'
+
+curl -f -L "${DOWNLOAD_URL}/${VERSION}/dist.zip" -o ./dist.zip
```

Comment:
check existence of the downloaded (extracted) content, do nothing if it's already downloaded.